docs(character): Add warning about missing feature (#2754)
starship/starship: docs/config/README.md

look at [this example](#with-custom-error-shape).

::: warning

`error_symbol` is not supported on elvish shell.

:::

::: warning

> `vicmd_symbol` is not supported for Bash's vi mode.

Reviewer: I think it might be better to just list the shells that are supported. For instance the PowerShell vi mode isn't supported either.

Reviewer 2: TIL Powershell has vi mode

> `vicmd_symbol` is not supported for Bash's vi mode.

Author: ```suggestion
`vicmd_symbol` is only supported in fish and zsh. 
```
